rpms/pulp-docker/pulp-docker.spec

```diff
@@ -0,0 +1,142 @@
+%global required_pulp_version 2.8.0
+
+Name: pulp-docker
+Version: 2.0.0
+Release: 1%{?dist}
+
+License:   GPLv2
```

Comment:
You are correct, the COPYRIGHT file does say that you can pick a later version of the GPL. Good catch!